### ISSUE TITLE
Please release 0.4.24 version with the security fix of 0.5.0 - important

### ISSUE THREAD

**Issue by lorand-horvath** [Leonidas-from-XIV/node-xml2js#671]
@Leonidas-from-XIV There has been a stream of issues coming in mainly from JIMP users due to the transient dependency on xml2js via `^0.4` which doesn't bump up to the fixed version 0.5.
Please see https://github.com/mattdesl/parse-bmfont-xml/pull/4 and https://github.com/jimp-dev/jimp/issues/1223 for more details.

The problem is that @mattdesl the developer of parse-bmfont-xml is unreachable.
So a quick fix to these problems would be for you to release a `xml2js@0.4.24` version (with the same content as 0.5.0) which would then be picked up when reinstalling JIMP via transient parse-bmfont-xml.

Thanks a lot!

**Comment by Leonidas-from-XIV** [comment]
I don't think I can do that because instead of keeping some packages at old versions (which can be overridden using npm and depending on the package might work) it would potentially break every single dependency.

So the best I can suggest is to either override it (if possible, given some changes are indeed backwards incompatible) or fork parse-bmfont-xml and update the code.

**Comment by Leonidas-from-XIV** [comment]
Yes, in this case it works but that's not true of all the packages that depend on xml2js unfortunately.

**Comment by lorand-horvath** [comment]
Fixed in https://github.com/mattdesl/parse-bmfont-xml/pull/4
